### ISSUE TITLE
coalang: Let Bears have their own coalang files.

### ISSUE THREAD

**Issue by abhsag24** [coala-analyzer/coala#2367]
so every bear has their own coalang with settings  that are obscure or not used by any other bear, like `encapsulators` for the `IndentationBear`, these settings get appended on top of the settings of the regular coalang

**Comment by Makman2** [comment]
I thought we want to merge each coalang?

**Comment by abhsag24** [comment]
merge each coalang?

**Comment by Makman2** [comment]
Didn't you say recently we want some kind of manager, where each language gets its own coalang file? And then you only use the manager to retrieve the right settings :O

**Comment by abhsag24** [comment]
each language already has their own coalag file?

**Comment by Makman2** [comment]
yeah but they are separated between each module that uses different information regarding that language. And I thought it was the goal to unite these files ^^
